content/java/concepts/strings/terms/equalsIgnoreCase/equalsIgnoreCase.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: '.equalsIgnoreCase()'
+Description: 'This method compares strings.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'String'
+CatalogContent:
+  - 'learn-java'
+---
+
+The `equalsIgnoreCase` method compares the strings in the program and then gives an output of true or false. It is not a case sensitive method, so this method does not consider this as a factor while comparing the strings.
+
+## Syntax
+
+```pseudo
+String nameOfYourString = "the value you assign";
+String nameOfYourString2 = "THE VALUE YOU ASSIGN";
+String nameOfYourString3 = "Value you assing";
+System.out.pritln(nameOfYourString.equalsIgnoreCase(nameOfYourString2)); //output of this will be true.
+System.out.pritln(nameOfYourString.equalsIgnoreCase(nameOfYourString3)); //output of this will be false.
+```
+
+## Example
+
+The example below will show you how this method works.
```

Comment:
The intro to the example should be more descriptive of what it does. And when the example produces output, we generally follow it with a ```shell block showing the output, usually introduced with a line like: "This example results in the following output."